src/plugin_manager/Plugin.cpp

```diff
@@ -50,10 +51,12 @@ Plugin::Plugin() : name_("Plugin"), parent_(std::make_shared<Entity>()),
                    transform_(std::make_shared<State>()),
                    id_to_team_map_(std::make_shared<std::unordered_map<int, int>>()),
                    id_to_ent_map_(std::make_shared<std::unordered_map<int, EntityPtr>>()),
-                   time_(std::make_shared<const Time>()) {
+                   time_(std::make_shared<const Time>()),
+                   param_server_(std::make_shared<ParameterServer>()) {
 }
 
-Plugin::~Plugin() {}
+Plugin::~Plugin() {
+}
```

Comment:
do we need this extra line?

Reply:
changed during a debug, i'll change back